### PR TITLE
Chirp-daily

### PR DESCRIPTION
**Pull request by johnsockwell**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download chirp-daily` is error-free.
- [x] `brew cask audit --download kk7ds-python-runtime` is error-free.
- [x] `brew cask style --fix chirp-daily` left no offenses.
- [ ] `brew cask style --fix kk7ds-python-runtime` left no offenses.
    - Unresolved `C: 13: 13: Align the elements of a hash literal if they span more than one line.`
- [x] `brew cask install chirp-daily` worked successfully.
- [x] `brew cask uninstall chirp-daily` worked successfully.
- [x] `brew cask install kk7ds-python-runtime` worked successfully.
- [x] `brew cask uninstall kk7ds-python-runtime` worked successfully.

### CODE REVIEW

File: Casks/kk7ds-python-runtime.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  url 'http://www.d-rats.com/download/OSX_Runtime/KK7DS_Python_Runtime_R10.pkg'
+  name 'KK7DS Python Runtime'
+  homepage 'http://www.d-rats.com/download/OSX_Runtime/'
+  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  pkg 'KK7DS_Python_Runtime_R10.pkg'
+
+  uninstall pkgutil: 'com.danplanet.python_runtime',
+            delete: '/opt/kk7ds'
```
Comment (vitorgalvao) [inline Casks/kk7ds-python-runtime.rb]: Put one more space between `delete:` and `'/opt/kk7ds'`, and try the `style` command again. It should be fixed.

File: Casks/kk7ds-python-runtime.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'kk7ds-python-runtime' do
+  version '10.1'
+  sha256 '5cee8acb941e39f93a4df6a99ed29a14c48da0bc5beb3b31068852b1fad8b009'
+
+  url 'http://www.d-rats.com/download/OSX_Runtime/KK7DS_Python_Runtime_R10.pkg'
```
Comment (vitorgalvao) [inline Casks/kk7ds-python-runtime.rb]: `url "http://www.d-rats.com/download/OSX_Runtime/KK7DS_Python_Runtime_R#{version.major}.pkg"`.

File: Casks/kk7ds-python-runtime.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  name 'KK7DS Python Runtime'
+  homepage 'http://www.d-rats.com/download/OSX_Runtime/'
+  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  pkg 'KK7DS_Python_Runtime_R10.pkg'
```
Comment (vitorgalvao) [inline Casks/kk7ds-python-runtime.rb]: `pkg "KK7DS_Python_Runtime_R#{version.major}.pkg"`

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones. In this case, one of your casks is good and being holded up by the other.

Also, why not submit the `kk7ds` cask in the main repo.

**Comment by johnsockwell** [comment]
I'll submit kk7ds-python-runtime to the main repo and issue a new pull request.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request closed without merging**